opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/hystrix/HystrixCommandTest.java

```diff
@@ -0,0 +1,83 @@
+package io.opentracing.contrib.spring.cloud.hystrix;
+
+import com.netflix.hystrix.contrib.javanica.annotation.HystrixCommand;
+import io.opentracing.Span;
+import io.opentracing.contrib.spring.cloud.MockTracingConfiguration;
+import io.opentracing.contrib.spring.cloud.TestUtils;
+import io.opentracing.mock.MockSpan;
+import io.opentracing.mock.MockTracer;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertEquals;
+
+@SpringBootTest(classes = {MockTracingConfiguration.class}, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@RunWith(SpringJUnit4ClassRunner.class)
+public class HystrixCommandTest {
+
+    @RestController
+    static class HelloWorldController2 {
+        @RequestMapping("/hello2")
+        public String hystrixCall() {
+            return "Hello2";
+        }
+
+    }
+
+    @RestController
+    static class HelloWorldController {
+        @Autowired
+        MockTracer mockTracer;
+
+        @Autowired
+        private TestRestTemplate restTemplate;
+
+        @RequestMapping("/hello")
```

Comment:
nit: give it a better mapping name. When looking at rest template invocation hello does not tell me anything.